### ISSUE TITLE
How to finetune ResMLP on CIFAR-10 and CIFAR-100?

### ISSUE THREAD

**Issue by yysung1123** [facebookresearch/deit#117]
Hi, could you provide the hyperparameters for reproduce results on the paper of ResMLP-12 and ResMLP-24 on CIFAR-10 and CIFAR-100?

**Comment by TouvronHugo** [comment]
Hi @yysung1123,
The hparams for CIFAR-10 and CIFAR-100 are:
- global batch size: 768
-  lr: 5e-6 
- epochs: 1000 
- weight-decay: 0.05  
- sched cosine  
- Repeated Aug: True
- Label smoothing: 0.1 
- warmup epochs: 5  
- RandAugment: rand-m9-mstd0.5-inc1 
-  Mixup: .8  
- Cutmix: 1.0 
- optimizer: Adamw 
- Warmup LR: 1e-6 
- drop-path: 0.1 (for 24) / 0.05 (for 12)
- Remove mixed precision: True

Best,

Hugo

**Comment by yysung1123** [comment]
Hi @TouvronHugo,
Thanks for your reply!
Should min-lr be default value (1e-5) or some value less than lr (5e-6)?

**Comment by TouvronHugo** [comment]
Hi @yysung1123,
We have not changed the default value for min-lr. 
But it is true that it could be interesting to make tests by decreasing it.
Best,
Hugo

**Comment by calvinmccarter-at-lightmatter** [comment]
Did you use distillation for CIFAR?

**Comment by yysung1123** [comment]
Hi @calvinmccarter-at-lightmatter 
I can reproduce the results on paper (ResMLP24: 89.5, ResMLP12: 87.0) without any distillation technique.

**Comment by shadowpa0327** [comment]
@calvinmccarter-at-lightmatter  I have rebuild the result using the parameter given by @TouvronHugo without using the distillation. Hope this will help.
